test_runner/scenario_test_runner/test/scenario/autoware-simple.yaml

```diff
@@ -85,7 +85,7 @@ OpenSCENARIO:
                       dynamicsShape: step
                     SpeedActionTarget:
                       AbsoluteTargetSpeed:
-                        value: 2.778
+                        value: 0.0
```

Comment:
@pijaro Add comment here please

Reply:
Done

Reply:
The feature that gives Autoware a non-zero initial velocity is provided in case we want to test a scenario at cruising speed but cannot provide a map with straight lines that Autoware can accelerate sufficiently.
This is currently achieved in a very ad hoc way, but we are aware that it can adversely affect Autoware's planner.

If you have problems with the .Auto planner, you can temporarily turn this feature off.
In that case, instead of changing the scenario, fix it so that EgoEntity :: setTargetSpeed does nothing when it is .Auto.

Reply:
Thank you for the explanation. We will change it as you suggested.